examples/source/interactivity-dynamic-content/Poll/index.html

```diff
@@ -80,7 +80,7 @@
       -->
       <form id="sample-poll"
             method="post"
-            action-xhr="<% hosts.backend %>/samples_templates/poll/submit"
+            action-xhr="submit"
```

Comment:
nit: change to `./submit` to make it clear that this is an URL. Otherwise it looks a bit confusing. Even better: rename to endpoint to `submit-poll`.

Reply:
Makes it less confusing. Thanks for the feedback!

Reply:
This might've caused this #2671